Vungle/VungleRouter.m

```diff
@@ -976,6 +875,70 @@ - (void)vungleWillLeaveApplicationForPlacementID:(NSString *)placementID
     [targetDelegate vungleAdWillLeaveApplication];
 }
 
+#pragma mark - VungleSDKHBDelegate Methods
+
+- (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
+                          eventID:(nullable NSString *)eventID
+                            error:(nullable NSError *)error
+{
+    if (!eventID.length) {
+        return;
+    }
+    [self vungleAdPlayabilityUpdate:isAdPlayable placementID:nil eventID:eventID error:error];
+}
+
+- (void)vungleWillShowAdForEventID:(nullable NSString *)eventID
+{
+    if (!eventID.length) {
+        return;
+    }
+    [self vungleWillShowAdForPlacementID:nil eventID:eventID];
+}
+
+- (void)vungleDidShowAdForEventID:(nullable NSString *)eventID
+{
+    [self vungleDidShowAdForPlacementID:nil eventID:eventID];
+}
+
+- (void)vungleAdViewedForAdUnit:(nullable NSString *)eventID
+{
+    if (!eventID.length) {
+        return;
+    }
+    [self vungleAdViewedForPlacement:nil eventID:eventID];
+}
+
+- (void)vungleWillCloseAdForEventID:(nonnull NSString *)eventID
+{
```

Comment:
Same as above, might we need to check the `eventID.length` here like above ? Thanks a lot. ^_^

Reply:
Updated.